recipe/meta.yaml

```diff
@@ -20,18 +20,22 @@ requirements:
     - python >=3.8
     - pytest-runner >=5.2
   run:
-    - aicsimageio >=4.1
+    - aicsimageio >=4.6.3
     - fsspec
-    - napari >=0.4.11
-    - napari-plugin-engine >=0.1.4
+    - napari >=0.4.13
+    - psutil >=5.7.0
     - python >=3.8
+    # plugins
+    - bioformats_jar >=2020.5.27
+    - readlif >=0.6.4
+    - nd2 >=0.3.0
+    - mrc >=0.2.0
+
```

Comment:
notably no `imageio`: https://github.com/AllenCellModeling/aicsimageio/blob/main/setup.py#L33

Don't know if that is shipped with napari by default?

Reply:
or aicspylibczi but seemingly that is the one that is giving our conda builds problems.

Reply:
ah, sure, we can add imageio, (but yes, it will be in napari too)

Reply:
> or aicspylibczi but seemingly that is the one that is giving our conda builds problems.

correct, if we added that here, this recipe would fail to build, since there is no aicspylibczi on conda-forge